src/bin/edit/localization.rs

```diff
@@ -566,6 +598,7 @@ const S_LANG_LUT: [[&str; LangId::Count as usize]; LocId::Count as usize] = [
         /* it      */ "Modifiche non salvate",
         /* ja      */ "未保存の変更",
         /* ko      */ "저장되지 않은 변경 사항",
+        /* pl      */ "Masz nie zapisane zmianny",
```

Comment:
```suggestion
        /* pl      */ "Masz niezapisane zmiany",
```
(double 'n' removed, join notation for participle)